audio/out/ao_coreaudio_utils.h

```diff
@@ -48,8 +48,10 @@ bool check_ca_st(struct ao *ao, int level, OSStatus code, const char *message);
         if (err != noErr) goto label; \
     } while (0)
 
+#if HAVE_COREAUDIO
 void ca_get_device_list(struct ao *ao, struct ao_device_list *list);
```

Comment:
No need to ifdef symbols that can be declared unconditionally, even if they're not always defined.

Also, no device selection on iOS?